src/HeadProvider.js

```diff
@@ -12,13 +14,19 @@ export default class HeadProvider extends React.Component {
 
   state = {
     list: [],
-    addClientTag: tag => {
+    addClientTag: (tag, name) => {
       this.setState(state => ({
-        list: [...state.list, tag],
+        list: [...state.list, `${tag}:${name}`],
       }));
       this.index += 1;
       return this.index;
     },
+    shouldRenderTag: index => {
+      const { list } = this.state;
+      const id = list[index];
+      const [tag] = id.split(':');
+      return cascadingTags.includes(tag) && list.lastIndexOf(id) === index;
```

Comment:
No `includes` in IE11 :( -- I guess it will be polyfilled but maybe better off just using `.indexOf`